tests-web/www/js/layers.js

```diff
@@ -1,7 +1,28 @@
 export const makeJsonLayer = (text) => {
-    return new L.geoJSON(JSON.parse(text), { style: { color: '#f55' }});
+    return new L.geoJSON(JSON.parse(text), { style: styleGeoJson });
 };
 
+const intersectionColours = {
+    undefined: '#666', // for default tarmac
+    Connection: '#666',
+    MultiConnection: '#669',
+    Merge: '#969',
+    Crossing: '#966',
+    Terminus: '#999',
+    MapEdge: '#696',
+}
```

Comment:
![image](https://user-images.githubusercontent.com/3784591/181663882-dc915dba-e9d0-430e-982d-77a18f66a0e2.png)

Here, non-intersections get the value for the string key `'undefined'`, because they have no `complexity`